src/overlays/actors/ovl_Dm_Tsg/z_dm_tsg.h

```diff
@@ -5,9 +5,18 @@
 
 struct DmTsg;
 
+typedef void (*DmTsgActionFunc)(struct DmTsg*, PlayState*);
+
 typedef struct DmTsg {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x7DC];
+    /* 0x144 */ DmTsgActionFunc ActionFunc;
+    /* 0x148 */ Vec3f unk_148[100];
+    /* 0x5F8 */ f32 unk_5F8[100];
+    /* 0x788 */ f32 unk_788;
+    /* 0x78C */ s16 unk_78C[100];
+    /* 0x854 */ s16 unk_854;
+    /* 0x856 */ s16 unk_856[100];
```

Comment:
Seems like all of those arrays should always have the same size by looking at the code.
Could you make a define for the size (like `#define DMTSG_UNK_SIZE 100`) and use it everywhere?
```suggestion
    /* 0x148 */ Vec3f unk_148[DMTSG_UNK_SIZE];
    /* 0x5F8 */ f32 unk_5F8[DMTSG_UNK_SIZE];
    /* 0x788 */ f32 unk_788;
    /* 0x78C */ s16 unk_78C[DMTSG_UNK_SIZE];
    /* 0x854 */ s16 unk_854;
    /* 0x856 */ s16 unk_856[DMTSG_UNK_SIZE];
```